sonic_data_client/virtual_db.go

```diff
@@ -149,7 +149,10 @@ func getPfcwdMap() (map[string]map[string]string, error) {
 		}
 
 		for _, key := range resp {
-			name := key[13:]
+			if strings.Contains(key, "GLOBAL") || strings.Contains(key, "global") { // ignore PFC_WD|global / PFC_WD|GLOBAL
+				continue
+			}
+			name := key[7:]
```

Comment:
instead of "7", should we get the length of "PFC_WD|" using a function for future proof?

Reply:
Yes, I can make that change.